pyproject.toml

```
@@ -11,9 +11,9 @@ python = "^3.12"
 Celery = "==5.2.7"
 crispy-forms-gds = "^0.2.3"
 dataclasses-json = "==0.5.7"
-django = "^4.1.10"
+django = "^4.2"
```

Can we specify 4.2.11 here as this is the minimum security version we are willing to run
```suggestion
django = "^4.2.11"
```